### ISSUE TITLE
feature suggestion: Image preview

### ISSUE THREAD

**Issue by qiansen1386** [M2Team/NanaZip#72]
![image](https://user-images.githubusercontent.com/1759658/145568598-771e07b8-2526-42d7-a0f5-52c8c91745ba.png)

![image](https://user-images.githubusercontent.com/1759658/145569514-dde8067b-6330-4d4f-819e-eed0005a605b.png)

Possible to unzip the selected image in the background and show it in a side panel? 
Or directly show thumbnails of all the images. In this way, If the folder contains tons of images we must only unzip the ones in current windows.

**Comment by SakuraNeko** [comment]
We have received your suggestion but may need to evaluate it further before deciding whether to add it to our road map.

Sakura Neko

**Comment by SakuraNeko** [comment]
@qiansen1386 :

We have archived your suggestions (see: #64 ) and will add them to our roadmap at the proper time. We will close this issue and complete the archive.

Sakura Neko

**Comment by AndromedaMelody** [comment]
- Reopen this issue (archived in discussion[#64](https://github.com/M2Team/NanaZip/discussions/64)).
- For more information, please read #274. Thank you.

**Comment by Ivoz** [comment]
Yes, this is why I still use Bandizip currently. I would love to find another archiver that implement this feature, so I can move away from Bandizip. It's very handy to be able to look through a zip of images instead of having to extract them first.
